coralproject/talk
# Slovak translation

## What does this PR do?

Adds partial Slovak (sk-SK) translations to the project.

## How do I test this PR?

Switch to Slovak language and manually check if translatable strings are translated.
